.github/workflows/ci.yml

```
@@ -0,0 +1,115 @@
+name: CI
+
+on:
+  pull_request:
+  push:
+    # Filtering branches here prevents duplicate builds from pull_request and push
+    branches:
+      - master
+      - beta
+      - 'v*'
+      - /^greenkeeper.*$/
+
+    # Always run CI for tags
+    tags:
+      - '*'
+
+  # Early issue detection: run CI weekly on Sundays
+  schedule:
+  - cron:  '0 6 * * 0'
+
+env:
+  CI: true
+
+jobs:
+  test-mocha:
+    name: Mocha Tests
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 10.x]
```

We can avoid duplicate job definitions by using a matrix for OS also:

```suggestion
  test-mocha:
    name: "Mocha - ${{ matrix.node-version }} - ${{ matrix.os }}"
    runs-on: "${{ matrix.os }}-latest"

    strategy:
      matrix:
        node-version: [12.x, 10.x]
        os: [ubuntu, windows]
```